src/redux/sagas/utils/getNetworkClient.ts

```diff
@@ -25,7 +25,7 @@ const getNetworkClient = async () => {
 
   const reputationOracleUrl = process.env.REPUTATION_ORACLE_ENDPOINT
     ? new URL(process.env.REPUTATION_ORACLE_ENDPOINT)
-    : new URL(`/reputation`, window.location.origin);
```

Comment:
This actually not a good change, since we were using `window.location.origin` as a fallback, which is still valid.

You still have to pass in a env variable `REPUTATION_ORACLE_ENDPOINT` to it, so if one's not set, chances are, the second is not set either.

Also, this is just a dev only use